commit 4a3757e0ae269f710292dd75013532c5a57ccb00
Author: Matthew Garrett <mjg@redhat.com>
Date:   Thu Sep 16 13:38:38 2010 -0400

    uvc: Enable USB autosuspend by default on uvcvideo
    
    We've been doing this for a while in Fedora without any complaints.
    
    Signed-off-by: Matthew Garrett <mjg@redhat.com>

diff --git a/drivers/media/video/uvc/uvc_driver.c b/drivers/media/video/uvc/uvc_driver.c
index 8bdd940..28ed5b4 100644
--- a/drivers/media/video/uvc/uvc_driver.c
+++ b/drivers/media/video/uvc/uvc_driver.c
@@ -1814,6 +1814,7 @@ static int uvc_probe(struct usb_interface *intf,
 	}
 
 	uvc_trace(UVC_TRACE_PROBE, "UVC device initialized.\n");
+	usb_enable_autosuspend(udev);
 	return 0;
 
 error:
